q2_fondue/plugin_setup.py

```diff
@@ -131,6 +132,8 @@
     parameter_descriptions={
         **common_param_descr,
         'retries': 'Number of retries to fetch sequences (default: 2).',
+        'restricted_access': 'If sequence fetch requires dbGaP repository '
+        'key (default=False).'
```

Comment:
```suggestion
        'restricted_access': 'If sequence fetch requires dbGaP repository key.'
```

No need to insert default values here - q2cli renders those automatically.

Reply:
I guess you could also remove it from the `retries`, now that I'm looking at it.